deployment/terraform/trace-reader/templates/trace-reader.conf

```diff
@@ -5,13 +5,15 @@ service {
       cert.path = ""
       private.key.path = ""
     }
+    max.message.size = 52428800 # 50MB in bytes
 }
 
 backend {
   client {
     host = "localhost"
     port = 8090
   }
+  max.message.size = 52428800 # 50MB in bytes
```

Comment:
does the grpc client also need this config?

Reply:
Actually this property is a no-op. We can remove it . This was done to keep deployment config and base config in sync.

Reply:
Remove it from both the places.

Reply:
Yup, I was on it. It is done. Let the build run